docs/python_support/reference.md

```diff
@@ -598,11 +598,11 @@ For a graph client can:
 
 - request status (gRPC and REST)
 - request metadata (gRPC and REST)
-- request inference (gRPC)
+- request inference (gRPC and REST)
 
 Learn more about how [MediaPipe flow works in OpenVINO Model Server](../mediapipe.md)
 
-For inference, if the format of graph input stream is [`OvmsPyTensor`](https://docs.openvino.ai/nightly/ovms_docs_python_support_reference.html#graph-input-and-output-streams), then the data in the KServe request must be encapsulated in either `ModelInferRequest`'s [InferTensorContents](https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/grpc_predict_v2.proto#L155) or [raw_input_contents](https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/grpc_predict_v2.proto#L202). If the graph has a `OvmsPyTensor` output stream, then the data in the KServe response can be found in `raw_output_contents` field (even if data in the request has been placed in `InferTensorContents`).
+For inference, data can be send both via [gRPC API](https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/required_api.md#grpc) and [KServe API](https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/required_api.md#httprest)(Only for unary calls). If the graph has a `OvmsPyTensor` output stream, then the data in the KServe response can be found in `raw_output_contents` field (even if data in the request has been placed in `InferTensorContents`).
```

Comment:
I think we should refactor this line to something like:

For details on how to interact with servable on mentioned endpoints see [KServe API specification](https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/required_api.md). 
While the request is created by the user and is accepted as long as it meets API requirements, the response structure user should expect in return is determined by a few factors:

**gRPC**:
- <some grpc specific stuff, like the fact that we put data in raw_output_contents>
- ...

**REST**:
 - <some rest specific stuff, like what data looks like depending on datatype in pytensor, when binary extension is used etc.>
- ...